src/test/java/org/javarosa/xpath/test/XPathEvalTest.java

```diff
@@ -58,8 +58,8 @@ private void testEval(String expr, FormInstance model, EvaluationContext ec, Obj
             if (exceptionExpected) {
                 fail("Expected exception, expression : " + expr);
             } else if ((result instanceof Double && expected instanceof Double)) {
-                Double o = (Double)result;
-                Double t = (Double)expected;
+                Double o = (Double) result;
```

Comment:
Agreed. For now I found how to configure AS to stop adding a space after casts, so will show you that today